Resources/Prototypes/Datasets/tips.yml

```diff
@@ -123,3 +123,47 @@
+  - "If your ship has a big crew, you can ask the SR for radio keys for your fellow crewmates. Some channels, such as Cargo or Science, are usually empty."
+  - "Before joining an ongoing round, you can see the list of ships registered on the New Frontiers. Perhaps some of them have jobs you would prefer?"
+  - "Don't forget to buy a medical implant before wandering into deep space! It will send a message to the local medics in the event of your death."
+  - "As a medic, you should cooperate with others when there's a medical emergency! It never helps when two medical ships arrive at the same emergency signal."
+  - "As a medic, you can ask the local SR for a medical PDA. It can double as a health analyzer!"
+  - "Medical ships often have access to the hypospray. An experienced chemist can turn it into a lethal weapon, or use it to safely put people to sleep."
+  - "If you're hurt, try to ask for medical help on the radio. Medics are often eager to heal the wounded, as their job usually gets boring after hours of waiting."
+  - "Always carry a GPS with yourself! It can help you locate your station, or tell your location to others and get saved."
+  - "Smuggling is very profitable, but also very risky! Remember to have an IFF console and hide your IFF before going for a dead drop."
+  - "As a smuggler, try to destroy any evidence of your crimes. Remember, your PDA has a notepad app, which can be used to store important notes in a digital form."
+  - "Security has a special IFF console that completely hides their ship on the radar! If you're an outlaw, watch out for these invisible prowlers!"
+  - "Certain ships can act as motherships. They have shipyard consoles, which allow you to purchase and sell other smaller ships!"
+  - "As a crewmemeber, you can start a mutiny if your captain refuses to pay you. Be careful though, as the captain has the permission to execute the mutiny leader!"
+  - "Your PDA has a pre-installed news application. Check it out, perhaps the local reporter posted some interesting news?"
+  - "There's a special app installed in your PDA, which allows you to see posted bounties. If you can fulfill one of them, you can ask the local SR or Sheriff for a reward!"
+  - "Remember, Frontier Outpost has a 200m wide safezone. It's forbidden to seriously harm people or property in any way within that limit!"
+  - "If you're using an IFF console, remember to turn your IFF on near frontier, or you'll risk facing security on your ship!"
+  - "Going on an expedition alone is very risky! Make sure to always have a crewmate to save you in case you get ambushed by a xeno queen."
+  - "Expeditions can yield rare and useful items, and not all of them are legal. Try not to leave those lying on the floor, as security can board you at any moment!"
+  - "Sometimes there's a Station Traffic Controller on the frontier outpost. Make sure to contact them before docking, or you may face a fine!"
```

Comment:
Could be useful to refer to the traffic encryption key in the survival box. Might be worth a separate tip entirely.